### PR TITLE
[feat]: Added View Reward and Claimed

### PR DESCRIPTION
**Pull request by red-game-dev**
Introduced new features for `QuestDetails`

- View Reward incl animation
- Claimed

### CODE REVIEW

File: src/components/QuestDetails/components/Reward/index.tsx [hunk 1]
```diff
@@ -40,8 +44,24 @@ export default function Reward({ reward, i18n }: RewardProps) {
...
+
+    window.open(reward.marketplaceUrl, '_blank')
+  }
+
   return (
     <div key={reward.title} className={styles.rewardContainer}>
+      {reward.isClaimed ? (
+        <div className={styles.isClaimed}>{i18n.claimed}</div>
+      ) : null}
+      <div className={styles.viewRewardContainer} onClick={onViewRewardClick}>
```
Comment (BrettCleary) [inline src/components/QuestDetails/components/Reward/index.tsx:61]: can you make this an a tag and set the src to `reward.marketplaceUrl`?

**Review by BrettCleary — APPROVED** [review]
looks good and is working. we need to make the view reward container an a tag though. Using the semantically correct tag allows users can copy paste the url, ctrl + click to open in a new tab, etc. which will be useful in the store

File: src/components/QuestDetails/components/Reward/index.tsx (hunk 1 above)
Comment (red-game-dev) [reply src/components/QuestDetails/components/Reward/index.tsx:61]: Yup can do 🙌

Comment (red-game-dev) [reply src/components/QuestDetails/components/Reward/index.tsx]: Sorted 🙌

**Pull request merged**